frontend/src/components/regions/UserActions/index.jsx

```diff
@@ -170,7 +119,10 @@ const mapStateToProps = (state) => {
       state.FETCH_RECRUIT_PROJECT_REVIEWS_PAGE,
     FETCH_USER_ACTIONS_CARDS_COMPLETED_PAGE:
       state.FETCH_USER_ACTIONS_CARDS_COMPLETED_PAGE,
+    FETCH_ACTIVITY_LOG_ENTRIES: state.FETCH_ACTIVITY_LOG_ENTRIES,
     userBurndownStats: state.apiEntities.burndownSnapshots || {},
```

Comment:
can i fix this in a different PR? its breaking the burndown chart when I change it

Reply:
@FaithMo This is a part of the render tidy but has not been prioritized as a task yet. However in the interest of Scout rules, I think you can go ahead and fix this render tidy. Should be a quick one and I think with @Ngoakor12 helping you out, you can hit it out of the park. This is a small exception since This one seems like a really small one to fix.

If you feel it is larger than I'm seeing at a glance then by all means, do it on another branch. If not, its a quick win.